data/aerial-photography/index.html

```diff
@@ -27,12 +28,12 @@ <h3 id="GOOGLE">
     </a>
     <ul class="dotless">
       <li>Natural Color (RGB)
-      <li>~6" pixels
+      <li>~6-inch pixels
     </ul>
-    <p>AGRC owns a public sector license to Google's statewide 6" aerial photography. Contractors and partners may also qualify. The Google license is for color aerial photography, typically collected within 3 years, from the spring, summer or fall. The imagery is statewide. The resolution is 6 inch or better with a horizontal positional accuracy to achieve or exceed one meter (C90) in most areas without significant vertical relief.</p>
+    <p>AGRC owns a public sector license to Google's statewide 6-inch aerial photography. Contractors and partners may also qualify. The Google license is for color aerial photography, typically collected within 3 years, from the spring, summer, or fall. The imagery is statewide, and the resolution is 6 inch or better with a horizontal positional accuracy to achieve or exceed 1 meter (C90) in most areas without significant vertical relief.</p>
```

Comment:
What do we mean with "collected within 3 years" ? Within 3 years of what? Not completely clear to me.

Reply:
let's completely remove this because it is not true anymore
, typically collected within 3 years,